examples/launchpad/README.md

```diff
@@ -0,0 +1,40 @@
+# LaunchPad
+
+Users of the Launchpad product will need to pass in certain headers in order to make API requests.
+
+## EdgeHeaders
+EdgeHeaders can be passed into request calls, the additional parameter is available in all reference client functions,
+get_aggs, and snapshots
```

Comment:
Even though it's only for these endpoints now, product asks us to enable it for all.